mungegithub/submit-queue/pv.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  labels:
+    app: submit-queue
+  name: submit-queue-github-cache
+spec:
+  capacity:
+    storage: 1Gi
+  accessModes:
+    - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Recycle
```

Comment:
explain 'erase on disconnect....    It would only get erased if you deleted the PVC (hint, don't delete the PVC)